CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
    * BREAKING CHANGE: minimum SDK constraint increased to 1.21.0. This allows
      use of async-await and generic function in Quiver.
    * BREAKING CHANGE: eliminated deprecated `FakeTimer`.
+   * BREAKING CHANGE: `StreamBuffer<T>` now implements `StreamConsumer<T>` as
```

Comment:
This is less of a breaking change and more of a "return to normal" that was prior to #296